server/src/main/java/org/elasticsearch/action/admin/cluster/bootstrap/GetDiscoveredNodesAction.java

```diff
@@ -23,7 +23,7 @@
 
 public class GetDiscoveredNodesAction extends Action<GetDiscoveredNodesResponse> {
     public static final GetDiscoveredNodesAction INSTANCE = new GetDiscoveredNodesAction();
-    public static final String NAME = "cluster:monitor/discovered_nodes";
+    public static final String NAME = "cluster:admin/get_bootstrap_discovered_nodes";
```

Comment:
maybe `cluster:admin/bootstrap/discover_nodes` and call the other one `cluster:admin/bootstrap/set_voting_config` and then whitelist `cluster:admin/bootstrap/*`?

Reply:
Thanks. 981dcc0434c8db3eff72ab9c50999710f13d4520.